### ISSUE TITLE
Support for Android TV

### ISSUE THREAD

**Issue by ehsannarmani** [jump-sdk/jetpack_compose_country_code_picker_emoji#51]
java.lang.AbstractMethodError: abstract method "void androidx.compose.foundation.lazy.layout.LazyLayoutItemProvider.Item(int, java.lang.Object, androidx.compose.runtime.Composer, int)"

**Comment by hbmartin** [comment]
Closing for now, feel free to reopen if the above solution does not work

**Comment by ehsannarmani** [comment]
yep, it's ok in 1.5.4, but just for mobile, in tv, we have focus problem.

**Comment by hbmartin** [comment]
Can you describe the problem in more detail or share a sample project? I have not used Android TV but happy to take a look.
